Harmony/Tools/AccessTools.cs

```diff
@@ -1156,11 +1157,22 @@ public static void RethrowException(Exception exception)
 			throw exception;
 		}
 
-		/// <summary>Tells you if the current runtime is based on Mono</summary>
-		/// <returns>True if we are running under Mono, false otherwise (.NET)</returns>
+		/// <summary>True if the current runtime is based on Mono, false otherwise (.NET)</summary>
 		///
 		public static bool IsMonoRuntime { get; } = Type.GetType("Mono.Runtime") != null;
 
+		/// <summary>True if the current runtime is .NET Framework, false otherwise (.NET Core or Mono, although latter isn't guaranteed)</summary>
+		///
+		public static bool IsNetFrameworkRuntime { get; } =
+			TypeByName("System.Runtime.InteropServices.RuntimeInformation")?.GetProperty("FrameworkDescription")
+			.GetValue(null, null).ToString().StartsWith(".NET Framework") ?? true;
```

Comment:
The default is `true`? Is that correct?

Reply:
Yes, because `System.Runtime.InteropServices.RuntimeInformation` only exists on .NET Core (any version) and .NET Framework 4.7.1+, so if it's missing, it should mean that some old .NET Framework runtime is being used.

Although, it's possible that an old Mono runtime (like that used in RimWorld 1.0 or other old Unity-based versions) also don't have `RuntimeInformation`, so I'll fix it so that that `true` is `IsMonoRuntime is false`.